fofix/game/song/song.py

```diff
@@ -325,20 +325,31 @@ def getScores(self, part):
             scores = self._get("scores", str, "")
             scores_ext = self._get("scores_ext", str, "")
 
+        if scores:
+            try:
+                scores = cerealizer.loads(binascii.unhexlify(scores))
+            except:
```

Comment:
You should specify the exception (use `Exception` instead).

Reply:
In order to put an exception there, I need to look up the current top of the exception class hiercarchy. But I really don't care what that is, I just want to trap everything. Thus the unqualified "except".

Reply:
Please, name your exception. As it is written in the doc, you can either specify the exception or just use `Exception` (this is what you need, if I understand your issue).

Reply:
Turns out you are correct, but one should not use Exception. Here's the reason:
https://docs.python.org/2/howto/doanddont.html#except